FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -345,16 +378,9 @@ get_options()
 		log_Msg "T1w Brain: ${p_t1w_brain}"
 	fi
 
-	if [ -z "${p_t2w_image}" ]; then
-		if [ -z "${p_existing_subject}" ]; then
-			log_Err "T2w Image (--t2w-image= or --t2=) required"
-			error_count=$(( error_count + 1 ))
-		else
-			p_t2w_image="NONE"  # Need something assigned as a placeholder for positional parameters
-		fi
-	else
-		log_Msg "T2w Image: ${p_t2w_image}"
-	fi
+
+	# NOTE: Check for T2w image is moved to check_mpp_compliance(), as missing T2w is allowed in LegacyStyleData MPP mode.
```

Comment:
This comment should probably say "moved upwards", as the check of the t2 variable is not inside a different file anymore.